src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -64,9 +80,113 @@ internal static string GetProtocolVersion(SafeSslHandle ssl)
         [DllImport(Libraries.CryptoNative)]
         internal static extern int SslRead(SafeSslHandle ssl, byte[] buf, int num);
 
+        [DllImport(Libraries.CryptoNative)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static extern bool IsSslRenegotiatePending(SafeSslHandle ssl);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int SslShutdown(SafeSslHandle ssl);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern void SslSetBio(SafeSslHandle ssl, SafeBioHandle rbio, SafeBioHandle wbio);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int SslDoHandshake(SafeSslHandle ssl);
+
+        [DllImport(Libraries.CryptoNative)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static extern bool IsSslStateOK(SafeSslHandle ssl);
+
         // NOTE: this is just an (unsafe) overload to the BioWrite method from Interop.Bio.cs.
         [DllImport(Libraries.CryptoNative)]
         internal static unsafe extern int BioWrite(SafeBioHandle b, byte* data, int len);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern SafeX509Handle SslGetPeerCertificate(SafeSslHandle ssl);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern SafeSharedX509StackHandle SslGetPeerCertChain(SafeSslHandle ssl);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int SslCtxUseCertificate(SafeSslContextHandle ctx, SafeX509Handle certPtr);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int SslCtxUsePrivateKey(SafeSslContextHandle ctx, SafeEvpPKeyHandle keyPtr);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int SslCtxCheckPrivateKey(SafeSslContextHandle ctx);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int BioCtrlPending(SafeBioHandle bio);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern void SslCtxSetQuietShutdown(SafeSslContextHandle ctx);
+
+        [DllImport(Libraries.CryptoNative, EntryPoint = "SslGetClientCAList")]
+        private static extern SafeSharedX509NameStackHandle SslGetClientCAListPrivate(SafeSslHandle ssl);
```

Comment:
The convention in other places (where I just followed what @stephentoub did) was the suffix `_private` vs `Private` (since that looks like it might be a reasonable method to target)